gtfs-flex/spec-reference.md

```diff
@@ -0,0 +1,177 @@
+##Flexible Public Transportation Services in GTFS
```

Comment:
To following the existing [Google transit repo format](https://github.com/google/transit), could you please move and rename this file to `/gtfs-flex/spec/en/reference.md`?